baselines/bigquery-storage/test/gapic-big_query_storage-v1beta1.ts.baseline

```diff
@@ -19,6 +19,7 @@
 import * as protosTypes from '../protos/protos';
 import * as assert from 'assert';
 import { describe, it } from 'mocha';
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Unrelated, but I would love to have a proper `import` here (I remember there were some types problems when we tried it).

Reply:
yes, Justin also brought this up in the other PR, import module will bring types in. Basically it's about mock methods like `client._innerApiCalls.echo`, while `_innerApiCalls` is private in `EchoClient` do we want to fix them now? Thank you!

Reply:
Can you spend an hour (but not more) on that? If we can make `innerApiCalls` public (I don't see any real harm about doing it) it might become possible to import and mock it properly. If you figure out you need more changes, then let's leave it as is.

A better alternative is to move unit tests to `sinon`, removing all the custom mocks :)  But this will take more time and we're probably not going to do it now.